### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by Hanuklee** [libAtoms/QUIP#667] [Q&A]
Hello, developers!

I'm installing the `quippy-ase` by following `pip install quippy-ase` (https://pypi.org/project/quippy-ase/).
Before downlad, I create the virtual environment with `python=3.9` using conda.
Then, I type `pip install quippy-ase`, but error occur as below:
`(quippy) C:\Users\USER>pip install quippy-ase
ERROR: Could not find a version that satisfies the requirement quippy-ase (from versions: none)
ERROR: No matching distribution found for quippy-ase`

How can I handle it??

**Comment by jameskermode** [comment]
We only have binary releases for macOS and Linux, not Windows. You'll have to compile yourself.

**Reply by Hanuklee** [reply]
Ah.. Thank you very much. I'll try to install in Linux system.
